src/lamp_py/tableau/jobs/filtered_hyper.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         remote_output_location: S3Location,
         processed_schema: pyarrow.schema,
         tableau_project_name: str,
-        rollup_num_days: int = 7,  # default this to a week of data
+        rollup_num_days: Optional[int] = 7,  # default this to a week of data
```

Comment:
we are generally moving towards the `int | None` syntax to eliminate importing of the Typing library unnecessarily. Lets not introduce them back in. 

```suggestion
        rollup_num_days: int | None = 7,  # default this to a week of data
```